policyengine_canada/parameters/gov/provinces/qc/tax/income/credits/children_activities/disabled_children/age_eligibility.yaml

```diff
@@ -0,0 +1,27 @@
+description: Age eligibility of disabled children for the Quebec children's activities tax credit.
+brackets:
+  - amount:
+      2022-01-01: 0
+    threshold:
+      2022-01-01: 0
+  - amount:
+      2022-01-01: 1
+    threshold:
+      2022-01-01: 5
+  - amount:
+      2022-01-01: 0
+    threshold:
+      2022-01-01: 18
+metadata:
+  type: single_amount
+  threshold_unit: year
+  amount_unit: /1
+  label: Quebec children's activities disabled child age eligibility
+  reference:
+    - title: Revenu Quebec - Tax Credit for Children's Activities - Line 462
+      href: https://www.revenuquebec.ca/en/citizens/income-tax-return/completing-your-income-tax-return/how-to-complete-your-income-tax-return/line-by-line-help/451-to-480-refund-or-balance-due/line-462/point-25/
+    - title: Quebec I-3 - Taxation Act, Part I, Book IX, Title III, Chapter III.1, DIVISION II.12.2, 1029.8.66.6. "eligible child"
```

Comment:
```suggestion
    - title: Quebec I-3 - Taxation Act, Part I, Book IX, Title III, Chapter III.1, DIVISION II.12.2, 1029.8.66.6. (e) "eligible child"
```

Reply:
similar for other yaml files if possible

Reply:
This provision is not under subsection (e), but is a separate "eligible child" regulation without number.(e) is the previous regulation "artistic, cultural, recreational or developmental activity". (e) is a subsection of the previous regulation "artistic, cultural, recreational or developmental activity"

Will also check others